composer.json

```diff
@@ -12,7 +12,8 @@
         "php": "^7.4",
         "twig/twig": "^2.8.1",
         "symfony/framework-bundle": "^3.4 || ^4.2",
-        "symfony/console": "^3.4 || ^4.2"
+        "symfony/console": "^3.4 || ^4.2",
+        "symfony/process": "^3.4 || ^4.4 || ^5.0"
```

Comment:
Is there any reason why we would need Symfony 5?

This branch can't work with eZ 3.x anyway (Which makes even Symfony 4.2 superfluous :D )

Reply:
I believe this was taken as a reference https://github.com/netgen/git-hooks/blob/2.1.3/composer.json :) 
But yeah, I guess @vesnacustic could only require 3.4 and 4.4 same as other symfony components (to be consistent)

Reply:
Removed 5.0 here https://github.com/netgen/site-generator-bundle/pull/3/commits/98f4926ea78b6fc352a9e46b253d3c24d350a4a4 :)